package.json

```
@@ -172,6 +172,7 @@
     "@types/scheduler": "0.16.2",
     "@types/semver": "^6.2.0",
     "@types/tmp": "0.2.0",
+    "@types/use-sync-external-store": "^0.0.6",
```

```suggestion
    "@types/use-sync-external-store": "0.0.6",
```

We need to use exact version